homu/init.sls

```diff
@@ -46,3 +46,11 @@ homu:
     - require:
       - pip: homu
       - file: /home/servo/homu/cfg.toml
+
+/var/homu:
+  file.directory:
+    - template: jinja
+    - user:  servo/servo
```

Comment:
I meant that the user should be `servo`, and the group should also be `servo`. `someuser/somegroup` is a shorthand for that.